cross-region-replication-mm2-converter/src/test/java/com/amazonaws/services/crossregion/schemaregistry/kafkaconnect/CrossRegionReplicationMM2ConverterTest.java

```diff
@@ -0,0 +1,400 @@
+import com.amazonaws.services.schemaregistry.exception.AWSSchemaRegistryException;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializationFacade;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializerImpl;
+import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants;
+import com.amazonaws.services.schemaregistry.utils.AvroRecordType;
+
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.errors.DataException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.services.glue.model.DataFormat;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
+/**
+ * Unit tests for testing RegisterSchema class.
+ */
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+
+public class CrossRegionReplicationMM2ConverterTest {
+    @Mock
+    private GlueSchemaRegistryDeserializationFacade deserializationFacade;
+    @Mock
+    private GlueSchemaRegistrySerializationFacade serializationFacade;
+    @Mock
+    private AwsCredentialsProvider credProvider;
+    @Mock
+    private GlueSchemaRegistryDeserializerImpl deserializer;
+    @Mock
+    private GlueSchemaRegistrySerializerImpl serializer;
+    private final static byte[] ENCODED_DATA = new byte[] { 8, 9, 12, 83, 82 };
+    private final static byte[] USER_DATA = new byte[] { 12, 83, 82 };
+    private final static Schema SCHEMA_REGISTRY_SCHEMA = new Schema("{}", "AVRO", "schemaFoo");
+    private static final String testTopic = "User-Topic";
+
+    byte[] genericBytes = new byte[] {3, 0, -73, -76, -89, -16, -100, -106, 78, 74, -90, -121, -5,
+            93, -23, -17, 12, 99, 10, 115, 97, 110, 115, 97, -58, 1, 6, 114, 101, 100};
+
+
+    byte[] avroBytes = new byte[] {3, 0, 84, 24, 47, -109, 37, 124, 74, 77, -100,
+            -98, -12, 118, 41, 32, 57, -66, 30, 101, 110, 116, 101, 114, 116, 97, 105, 110, 109, 101, 110,
+            116, 95, 50, 0, 0, 0, 0, 0, 0, 20, 64};
+
+    private CrossRegionReplicationMM2Converter converter;
+
+
+
+    @BeforeEach
+    void setUp() {
+        converter = new CrossRegionReplicationMM2Converter(deserializationFacade, serializationFacade, credProvider, deserializer, serializer);
+    }
+
+    /**
+     * Test for Mm2Converter config method.
+     */
+    @Test
+    public void testConverter_configure() {
```

Comment:
I didn't find a test that tests the public method `convert`. Am I missing something?